test/helpers/test/time.test.js

```diff
@@ -0,0 +1,65 @@
+const time = require('../time');
+const shouldFail = require('../shouldFail');
+
+const BigNumber = web3.BigNumber;
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+describe('time', function () {
+  beforeEach(async function () {
+    this.start = await time.latest();
+  });
+
+  describe('increase', function () {
+    it('increases time by a duration', async function () {
+      await time.increase(time.duration.hours(1));
+
+      const end = this.start + time.duration.hours(1);
+      (await time.latest()).should.be.within(end - 20, end + 20); // +-20 sec tolerance
```

Comment:
We can use [`closeTo`](https://www.chaijs.com/api/bdd/#method_closeto). Can we define the tolerance in a file-wide constant?

Also, I'm curious, experimentally, what did you find the actual time increase to be?

Reply:
Didn't know about `closeTo`, thanks!

The time tolerance comes from the last block (and therefore the timestamp) being mined in the previous test, and the time between tests is variable (in particular, the time between `ganache` being set-up and the first test running may be very large). 

I ended up making this more explicit by adding an `advanceBlock` call before calling `latest()`, but we should make sure the documentation explains this behavior properly.